src/js/angular/ttyg/controllers/ttyg-view.controller.js

```diff
@@ -250,26 +251,33 @@ function TTYGViewCtrl($rootScope, $scope, $http, $timeout, $translate, $uibModal
         }
     };
 
-    const onCreateNewChat = (chatQuestion) => {
-        let newConversationId = undefined;
-        TTYGService.createConversation($scope.chatQuestion)
-            .then((conversationId) => {
-                newConversationId = conversationId;
-                return loadChats();
-            })
-            .then(() => {
+    /**
+     * @param {ChatItemModel} catItem
+     */
+    const onCreateNewChat = (catItem) => {
+        TTYGService.createConversation(catItem)
+            .then((chatId) => chatId)
+            .then((newChatId) => {
                 TTYGContextService.emit(TTYGEventName.CREATE_CHAT_SUCCESSFUL);
-                TTYGContextService.selectChat(TTYGContextService.getChats().getChat(newConversationId));
+                TTYGContextService.selectChat(TTYGContextService.getChats().getChat(newChatId));
             })
+            .then(loadChats)
```

Comment:
move load chats in previous then and remove this one

Reply:
Done